libs/sdk-react-native/ios/BreezSDKMapper.swift

```diff
@@ -455,7 +511,22 @@ class BreezSDKMapper {
             "message": messageSuccessActionData.message
         ]
     }
-    
+
+    static func dictionaryOf(receivePaymentResponse: ReceivePaymentResponse) -> [String: Any?] {
+        return [
+            "lnInvoice": dictionaryOf(lnInvoice: receivePaymentResponse.lnInvoice),
+            "openingFeeParams": dictionaryOf(params: receivePaymentResponse.openingFeeParams),
```

Comment:
Will cause problems when openingFeeParams is nil. Either nil check or change `dictionaryOf(openingFeeParams: OpeningFeeParams)` to handle a nil

Reply:
I changed the `dictionaryOf` to handle a nil from

```swift
static func dictionaryOf(openingFeeParams: OpeningFeeParams) -> [String: Any] 
```
to
```swift
static func dictionaryOf(openingFeeParams: OpeningFeeParams) -> [String: Any?] 
```

Reply:
I think the params should be optional
```suggestion
static func dictionaryOf(openingFeeParams: OpeningFeeParams?) -> [String: Any]?
```
and return a `nil` if openingFeeParams is nil

Reply:
Done in a9f7d3ca76bb568f7607a675dc3bb058e665b5b3